pages/_plugins/_capitalize_all.rb

```diff
@@ -0,0 +1,13 @@
+require 'liquid'
+require 'uri'
+
+# Capitalize all words of the input
+module Jekyll
+  module CapitalizeAll
+    def capitalize_all(words)
+      return words.split(' ').map(&:capitalize).join(' ')
+    end
+  end
+end
+
+Liquid::Template.register_filter(Jekyll::CapitalizeAll)
```

Comment:
IMHO, I don't think we want this. Data.gov has historically over-used capitalization but I think we should steer toward 18f's guidance which says not to [capitalize headings](https://content-guide.18f.gov/our-style/capitalization/#headings). The 10x folks designed r.d.g without title-case, so this introduces some bugs. For example, the document title is still sentence cased while the headings are title-cased https://resources-staging.data.gov/standards/.

If the category is in fact a title, then the category.name should be capitalized, and there would be no need for this filter and it would be cased consistently throughout the site.